SharedUtils/src/main/java/com/braintreepayments/api/HttpResponseCallback.kt

```diff
@@ -0,0 +1,14 @@
+package com.braintreepayments.api
+
+import androidx.annotation.MainThread
+import androidx.annotation.RestrictTo
+
+/**
+ * @suppress
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface HttpResponseCallback {
+
+    @MainThread
+    fun onResult(responseBody: String?, httpError: Exception?)
+}
```

Comment:
Had to convert this to Kotlin for the tests to pass because of a [bug in mockk](https://github.com/mockk/mockk/issues/865).